v8go.cc

```diff
@@ -134,16 +134,38 @@ IsolateHStatistics IsolationGetHeapStatistics(IsolatePtr ptr) {
 
 /********** Context **********/
 
+// Print provides a basic interface to print to stdout.
+static void Print(const FunctionCallbackInfo<Value>& args) {
+  bool first = true;
+  auto iso = args.GetIsolate();
+  for (int i = 0; i < args.Length(); i++) {
+    HandleScope handle_scope(iso);
+    if (first) {
+      first = false;
+    } else {
+      printf(" ");
+    }
+    String::Utf8Value value(iso, args[i]);
+    printf("%s", *value);
+  }
+  printf("\n");
+  fflush(stdout);
```

Comment:
What is this function? I can't see it defined. To flush stdout you would normally do something like:
```
f := bufio.NewWriter(os.Stdout)
defer f.Flush()
f.Write(b)
```